assets/sass/components/dashboard/_googlesitekit-ReportMetric.scss

```
@@ -77,6 +77,12 @@
 	vertical-align: bottom;
 }
 
+.googlesitekit-pagespeed-report-metric__hint-text {
+	color: $c-neutral-n-500;
+	letter-spacing: 0.25px;
```

Let's use our SCSS variable for letter-spacing. Also, let's add a line-height value that is same as the referenced note.

```suggestion
	letter-spacing: $ls-s;
	line-height: $lh-body-lg;
```